app/presenters/auction_presenter.rb

```diff
@@ -89,20 +89,24 @@ def bid_count
     bids.size
   end
 
-  def starts_at
+  def formatted_start_time
     DcTimePresenter.convert_and_format(model.start_datetime)
```

Comment:
thoughts on putting `DcTimePresenter.convert_and_format` into a private method so we can re-use it for these three methods rather than calling it each time?